packages/babel-parser/test/fixtures/es2017/async-functions/invalid-escape-sequence-function-list/input.js

```diff
@@ -1 +0,0 @@
-(\u0061sync function() { await x })
```

Comment:
Why this test is deleted? Is it covered by another test cases or do you rename it?

Reply:
I'll add this back, I think I accidentally deleted it by cleaning up unnecessary `options.json` files.